apps/web/src/components/BasicInformation/DiversityEquityInfo.tsx

```diff
@@ -0,0 +1,66 @@
+import { useIntl } from "react-intl";
+
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  getEmploymentEquityGroup,
+} from "@gc-digital-talent/i18n";
+
+import FieldDisplay from "~/components/FieldDisplay/FieldDisplay";
+
+export const DiversityEquityInfo_Fragment = graphql(/* GraphQL */ `
+  fragment DiversityEquityInfo on User {
+    isWoman
+    hasDisability
+    isVisibleMinority
+    indigenousCommunities {
+      value
+    }
+  }
+`);
+
+interface DiversityEquityInfoProps {
+  diversityEquityInfoQuery: FragmentType<typeof DiversityEquityInfo_Fragment>;
+}
+
+const DiversityEquityInfo = ({
+  diversityEquityInfoQuery,
+}: DiversityEquityInfoProps) => {
+  const intl = useIntl();
+  const { indigenousCommunities, hasDisability, isVisibleMinority, isWoman } =
+    getFragment(DiversityEquityInfo_Fragment, diversityEquityInfoQuery);
+
+  return (
+    <div
+      data-h2-display="base(grid)"
+      data-h2-grid-template-columns="p-tablet(1fr)"
+      data-h2-gap="base(x1)"
+    >
+      <FieldDisplay
+        data-h2-grid-column="base(span 2)"
+        label={intl.formatMessage(commonMessages.employmentEquity)}
+      >
+        <ul>
+          {!!indigenousCommunities?.length && (
+            <li>
+              {intl.formatMessage(getEmploymentEquityGroup("indigenous"))}
+            </li>
+          )}
+          {hasDisability && (
+            <li>
+              {intl.formatMessage(getEmploymentEquityGroup("disability"))}
+            </li>
+          )}
+          {isVisibleMinority && (
+            <li>{intl.formatMessage(getEmploymentEquityGroup("minority"))}</li>
+          )}
+          {isWoman && (
+            <li>{intl.formatMessage(getEmploymentEquityGroup("woman"))}</li>
+          )}
+        </ul>
```

Comment:
I think this should check to make sure at least one is true before outputting the `ul` so we don't end up with an empty one. Maybe a null message as well? :thinking:

Reply:
Great catch! Fixed in [Add null message if diversity quity info is empty](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13300/commits/1108ef57474a58af203d5c8890a7b1c9d3ca59c5)